language_service/src/completion.rs

```diff
@@ -432,3 +459,95 @@ impl Completion {
         }
     }
 }
+
+fn get_qasm_completions(
```

Comment:
Would be nice to pull the qasm-related stuff into a separate module like `qasm_completion.rs` - this is getting chunky